config-default.yml

```diff
@@ -367,7 +367,7 @@ anti_spam:
     rules:
         attachments:
             interval: 10
-            max: 9
+            max: 3
```

Comment:
This seems too strict to me. Most of filter trips are for people who send a lot of screenshots in help channels, or a lot of design images in the branding channel. Let's start with 6 and see how many false positives to false negatives we have.

Reply:
Fwiw the idea was to keep the current behavior but use the right filter when someone does spam images. I'm fine with changing the threshold though.